JobConfig/common/epilog.fcl

```diff
@@ -0,0 +1,5 @@
+# test values; these get overwritten by generate_fcl
+services.SeedService.baseSeed         :  773651
+services.SeedService.maxUniqueEngines :  20
+# by default, turn off histogram output
+services.TFileService.fileName : "/dev/null"
```

Comment:
Again, epilog setting are not "default" but force off with no way out.   Please avoid them.

Reply:
This is now a general production default.  The alternatives to epilog are repeating this in every file (error prone) or to override StandardServices, which Ray didn't like.

Reply:
Is this needed because we have code in production that makes histograms and it's too much work to track them all down?  I am pretty sure that if we did not have such code, this would not be needed.